subprojects/test-runner/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/configuration/ExperimentalExtension.kt

```diff
@@ -12,5 +12,11 @@ public abstract class ExperimentalExtension {
 
     public abstract val useInMemoryReport: Property<Boolean>
 
+    /**
+     * By default artifacts uploaded to file storage, and if file storage disabled you can't access it
+     * This is not intended behavior and should be changed
+     */
+    public abstract val saveTestArtifactsToOutputs: Property<Boolean>
```

Comment:
default value

Reply:
it defaults during validation phase, like other flags